source/arch/begin_run_event.json

```diff
@@ -0,0 +1,50 @@
+{
+    "properties": {
```

Comment:
Right! Ignore that comment. It was before we sorted out the project.type stuff below.